### PR TITLE
Add missing tools

### PR DESCRIPTION
**Pull request by demkodo**
-----
[View rendered activities/tool-management/1password.md](https://github.com/publiccodenet/about/blob/add-missing-tools/activities/tool-management/1password.md)
[View rendered activities/tool-management/miro.md](https://github.com/publiccodenet/about/blob/add-missing-tools/activities/tool-management/miro.md)
[View rendered activities/tool-management/voys.md](https://github.com/publiccodenet/about/blob/add-missing-tools/activities/tool-management/voys.md)

### CODE REVIEW

**Comment by demkodo** [comment]
I can't get this PR out. It keeps whining about Twitter in other sections. What is the best next step?

**Comment by ericherman** [comment]
This might help a bit: https://github.com/publiccodenet/jekyll-theme/pull/50

**Comment by Ainali** [comment]
Should they be added to an index as well?

File: activities/tool-management/miro.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Miro [has extensive documention.](https://help.miro.com/hc/en-us)
+
+## Miro management
+
+The Operations Coordinator, Chief Executive and Codebase Stewards can add new users to Miro.
+
+## Privacy and GDPR
+
+Miro's (rebranded from RealtimeBoard Inc.) Privacy Policy describes in detail how they use and collect information.
```
Comment (ElenaFdR) [inline activities/tool-management/miro.md:19]: Please add a hyperlink to the source URL for this text.

**Review by ElenaFdR — DISMISSED** [review]
This looks fab!

**Review by ElenaFdR — APPROVED** [review]

**Pull request merged**